src/components/social-share/styles.ts

```diff
@@ -0,0 +1,58 @@
+import { ThemeUICSSObject } from 'theme-ui';
+import theme from '@mdb/flora/theme';
+
+const SocialShareWrapper: ThemeUICSSObject = {
+  display: 'flex',
+  gridColumn: 'span 7',
+};
+
+const SocialShareIcon: ThemeUICSSObject = {
+  marginRight: ['14px', null, null, '16px'],
+};
+
+const CopyLinkWrapper: ThemeUICSSObject = {
+  position: 'relative',
+  button: {
+    outline: 'none',
+    border: 'none',
+    background: 'transparent',
+  },
```

Comment:
nitpick, but I think we should put `cursor: pointer` on this, just to keep in line with devcenter's version of this.